<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="skin_label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>481</width>
      <height>381</height>
     </rect>
    </property>
    <property name="text">
     <string>skin</string>
    </property>
   </widget>
   <widget class="QLabel" name="hair_label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>321</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>hair</string>
    </property>
   </widget>
   <widget class="QLabel" name="mood_label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>270</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>mood</string>
    </property>
   </widget>
   <widget class="QLabel" name="hat_label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>301</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>hat</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>191</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Skin </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="skin_box"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Hair</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="hair_box"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Hat</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="hat_box"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Mood</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="mood_box"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
